### ISSUE TITLE
Failure with Cargo.lock version = 4 (rustc 1.83.0)

### ISSUE THREAD

**Issue by daniestevez** [kbknapp/cargo-outdated#405]
I'm using the rustc 1.83.0 toolchain and finding the following problem with with `cargo-outdated`:
```
$ cargo outdated
error: failed to parse lock file at: /tmp/testcrate/Cargo.lock

Caused by:
  lock file version 4 requires `-Znext-lockfile-bump`
```

To replicate:
```
$ cargo new testcrate
$ cd testcrate
$ cargo update
$ cargo outdated
```

The `Cargo.lock` file looks like this:
```toml
# This file is automatically @generated by Cargo.
# It is not intended for manual editing.
version = 4

[[package]]
name = "testcrate"
version = "0.1.0"
```
Apparently version 4 is new in rust 1.83.0.

```
$ rustup show active-toolchain -v
stable-x86_64-unknown-linux-gnu (default)
rustc 1.83.0 (90b35a623 2024-11-26)
```

**Comment by oherrala** [comment]
This is already fixed in master branch after merge of #403.

**Comment by daniestevez** [comment]
Great. Marking as closed, then.
